user/reference/bionic.md

```diff
@@ -0,0 +1,167 @@
+---
+title: The Ubuntu 18.04 Build Environment
+layout: en
+---
+
+## What This Guide Covers
+
+This guide provides an overview of the packages, tools and settings available in the Bionic environment.
+
+## Using Bionic
+
+To route your builds to Ubuntu 18.04 LTS, add the following to your `.travis.yml`:
+
+```yaml
+dist: bionic
+```
+{: data-file=".travis.yml"}
+
+Please note that Bionic is available on our hosted fully virtualized
+infrastructure. If you are running an Enterprise installation, please reach out
+to [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com?subject=Try%20out%20Bionic) to see how you can use the Bionic Docker images.
+
+## Differences from the previous relealse images
+
+Travis CI Ubuntu 18.04 includes the following changes and improvements:
+
+### Third party apt-repositories removed
+
+While third party apt-repositories are used during the image provisioning, they are all removed from the Bionic build image. This has two benefits; a) reduced risk of unrelated interference and b) faster apt-get updates.
+
+To specify a third party apt-repository, you can [add the source with the apt addon](/user/installing-dependencies/#adding-apt-sources) and specify the packages. For example:
+
+```yaml
+dist: bionic
+group: dev
```

Comment:
@zhengpenghou when is the `group: dev` tag needed?

Reply:
@zhengphou @zhengpenghou ^^

Reply:
Yes. it's needed for now as a preview release.

Reply:
So then we need to add it everywhere we use `dist: bionic`, such as in the Snap pages I think? And we should probably explain on this page why it is there.

Reply:
If plan to use Ubuntu 18.04,  bionic is the one should be used to choose the right image.  This information has been updated in relevant pages.

Reply:
Right, but I'm still talking about `group: dev`.

Reply:
we have 3 group tags now, stable, edge and dev. @MariadeAnton your thoughts?

Reply:
Maria is away for another week. 
@zhengphou do you want to add a tiny note to this document detail when to use `dev` and see how we go from there?

Reply:
Done